src/algorithms/calorimetry/CalorimeterEoverPCut.cc

```diff
@@ -0,0 +1,56 @@
+#include <edm4eic/EDM4eicVersion.h>
+#if EDM4EIC_VERSION_MAJOR >= 8
```

Comment:
No need to guard this for older versions. This is mostly done for features that require data model evolution. Here you are using standard functionality only.